src/main/java/seedu/address/logic/commands/ExportCommand.java

```diff
@@ -0,0 +1,34 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_FILENAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_FOLDERNAME;
+
+import java.util.List;
+
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Exports single or multiple card folders into a .json file. Users must specify file name to export card folders to.
+ */
+public class ExportCommand extends Command {
+
+    public static final String COMMAND_WORD = "export";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": exports single or multiple card folders into"
+            + "a .json file.\n"
+            + "Parameters: "
+            + PREFIX_FOLDERNAME + "CARD_FOLDER_NAME [MORE CARD_FOLDER_NAMES]..."
+            + PREFIX_FILENAME + "Filename\n"
+            + "Example: " + COMMAND_WORD + "f/Human_anatomy f/Bone_structure n/myfilename";
```

Comment:
Shouldn't there be a .json at the end of myfilename?

Reply:
Ah i wanted to like abstract away the file structure from the user haha. What d u think ?

Reply:
> Please write your commit titles in the imperative next time
> i.e. So that it fits in the sentence: "This commit will <_commit message_>"

YES SENPAI !! >.<

Reply:
Yes, abstracting is an option, you can go that way. That would ensure that the data is always in json. Whichever the case, I think that the export and import commands should use the same format (import looks like it requires the .json).

i.e. As a user, if I do `import abc.json`, I would expect to be able to do `export abc.json` as well.

Reply:
Changed the format for import and export to always include .json format